include/proxy/http/HttpConfig.h

```diff
@@ -125,6 +125,7 @@ struct HttpStatsBlock {
   Metrics::Counter::AtomicType *misc_origin_server_bytes;
   Metrics::Counter::AtomicType *misc_user_agent_bytes;
   Metrics::Counter::AtomicType *missing_host_hdr;
+  Metrics::Counter::AtomicType *no_remap_404;
```

Comment:
Maybe better matching the ptr: "no_remap_matched" (404 here is a bit redundant).